src/plato-mini/.pdm.toml

```diff
@@ -0,0 +1,2 @@
+[python]
```

Comment:
Is this pdm-toml file needed in this PR? I'm worried that this toml file would break the reproducible of this PR.